source/SAMRAI/mesh/GriddingAlgorithm.h

```diff
@@ -509,7 +509,7 @@ class GriddingAlgorithm:
       return d_object_name;
    }
 
```

Comment:
Why is "private:" commented out here?

Reply:
I think it's because you can't have device lambdas in private class methods 😢

Reply:
@davidbeckingsale I'm returning to this far too late.  I don't see any device lambdas in this class.  Perhaps they were here in an earlier iteration of this branch, or am I missing something?  Do you think this private keyword can be reinstated?